### PR TITLE
[Package] Changing source directory name

### PR DESCRIPTION
**Pull request by benjaminye**
- Implements https://github.com/georgian-io/LLM-Finetuning-Hub/issues/101

### CODE REVIEW

File: llmtune/data/dataset_generator.py [hunk 1]
```diff
@@ -6,11 +6,8 @@
 
 import re
 from datasets import Dataset
-from rich.console import Console
-from rich.layout import Layout
-from rich.panel import Panel
```
Comment (truskovskiyk) [inline llmtune/data/dataset_generator.py:11]: why do we remove this in this PR

File: llmtune/finetune/lora.py [hunk 2]
```diff
@@ -23,10 +22,10 @@
 from rich.console import Console
 
 
-from src.pydantic_models.config_model import Config
-from src.utils.save_utils import DirectoryHelper
-from src.finetune.finetune import Finetune
```
Comment (truskovskiyk) [inline llmtune/finetune/lora.py:28]: why do we need this change?

File: llmtune/inference/lora.py [hunk 3]
```diff
@@ -4,20 +4,17 @@
 import csv
 
 from transformers import TextIteratorStreamer
-from rich.console import Console
-from rich.table import Table
-from rich.live import Live
```
Comment (truskovskiyk) [inline llmtune/inference/lora.py:9]: same question: I would keep renaming as one change and fixing rich as another PR

File: llmtune/qa/generics.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
 from abc import ABC, abstractmethod
 from typing import Union, List, Tuple, Dict
 import pandas as pd
-from src.ui.rich_ui import RichUI  
+from llmtune.ui.rich_ui import RichUI
 import statistics
-from src.qa.qa_tests import *
+from llmtune.qa.qa_tests import *
```
Comment (truskovskiyk) [inline llmtune/qa/generics.py:6]: please. do not do this. import only what you need

**Review by truskovskiyk — APPROVED** [review]

File: llmtune/qa/generics.py (hunk 4 above)
Comment (benjaminye) [reply llmtune/qa/generics.py:6]: - This has been resolved with Vivek
- We restructured the design pattern to eliminate having to import here at all

File: llmtune/inference/lora.py (hunk 3 above)
Comment (benjaminye) [reply llmtune/inference/lora.py:9]: Agreed

File: llmtune/finetune/lora.py (hunk 2 above)
Comment (benjaminye) [reply llmtune/finetune/lora.py:28]: Unused imports. But I agree this should be separate.

File: llmtune/data/dataset_generator.py (hunk 1 above)
Comment (benjaminye) [reply llmtune/data/dataset_generator.py:11]: Unused imports. But I agree this should be separate.

**Pull request merged**